packages/manager/modules/pci/src/projects/project/sidebar/sidebar.constant.js

```diff
@@ -113,55 +113,55 @@ export const MENU = [
         options: {
           state: 'pci.projects.project.users',
         },
-        translation: 'cloud_sidebar_project_management_users',
+        translation: 'Users',
       },
       {
         id: 'quota',
         options: {
           state: 'pci.projects.project.quota',
         },
-        translation: 'cloud_sidebar_project_management_quota_location',
+        translation: 'Quota and Localisation',
       },
       {
         id: 'ssh-keys',
         options: {
           state: 'pci.projects.project.sshKeys',
         },
-        translation: 'cloud_sidebar_project_management_ssh_keys',
+        translation: 'SSH Keys',
       },
       {
         id: 'billing',
         options: {
           state: 'pci.projects.project.billing',
         },
         regions: ['EU', 'CA'],
-        translation: 'cloud_sidebar_project_management_billing_control',
+        translation: 'Billing Control',
       },
       {
         id: 'vouchers',
         options: {
           state: 'pci.projects.project.vouchers',
         },
         regions: ['EU', 'CA'],
-        translation: 'cloud_sidebar_project_management_credit_vouchers',
+        translation: 'Credit and Vouchers',
       },
       {
         id: 'contacts',
         options: {
           state: 'pci.projects.project.contacts',
         },
         regions: ['EU', 'CA'],
-        translation: 'cloud_sidebar_project_management_contact_rights',
+        translation: 'Contacts and Rights',
       },
       {
         id: 'management-settings',
         options: {
           state: 'pci.projects.project.edit',
         },
-        translation: 'cloud_sidebar_project_management_settings',
+        translation: 'Project settings',
       },
     ],
-    translation: 'cloud_sidebar_project_management',
+    translation: 'Project Management',
```

Comment:
It's not really a `translation` anymore though, more like a `label` or `title`

Reply:
For sure. To be aligned with CSS selector I will prefer using `title` key.